lib/roo_on_rails/checks/github/branch_protection.rb

```diff
@@ -35,7 +36,9 @@ def fix
 
         def ensure_status_checks!
           status_checks = protection[:required_status_checks] || {}
-          fail! 'status checks do not include admins' unless status_checks[:include_admins]
+          enforce_admins = protection[:enforce_admins]
+          fail! 'status checks are not enforced for admins' unless enforce_admins &&
+                                                                   enforce_admins[:enabled]
```

Comment:
i'm probably missing something here, but enforce_admins is a boolean, so it will not respond to `#[]`